src/main/resources/view/EntityCard.css

```diff
@@ -0,0 +1,12 @@
+.cardPane{
+    -fx-effect: dropshadow(gaussian, rgba(0, 0, 0, 0.3), 10, 0.5, 0.0, 0.0);
+}
+
+.index, .id{
```

Comment:
Spacing between class name and curly brace

Reply:
Deleted this hahah